app/models/manageiq/providers/hawkular/middleware_manager/alert_manager.rb

```diff
@@ -27,17 +27,22 @@ def process_alert(operation, miq_alert)
 
     def convert_to_group_trigger(miq_alert)
       eval_method = miq_alert[:conditions][:eval_method]
-      firing_match = case eval_method
-                     when "mw_heap_used", "mw_non_heap_used" then 'ANY'
-                     else 'ALL'
-                     end
+      firing_match = 'ALL'
+      context = { 'dataId.type' => 'gauge', 'dataId.prefix' => 'hm_g_' }
```

Comment:
Context is there for reasons defined by the user, and you are the user :)  So, if you think it's useful that is OK by me.  I only wonder whether you should make a similar comment in the code, otherwise it will be totally unknown why there is unnecessary/purely informative context data hanging on the trigger.